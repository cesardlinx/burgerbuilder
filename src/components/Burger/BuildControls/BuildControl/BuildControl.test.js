import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import BuildControl from './BuildControl';

configure({ adapter: new Adapter() });

describe('<BuildControl />', () => {
  it('should render', () => {
    const wrapper = shallow(<BuildControl />);
    expect(wrapper).toMatchSnapshot();
  });
});