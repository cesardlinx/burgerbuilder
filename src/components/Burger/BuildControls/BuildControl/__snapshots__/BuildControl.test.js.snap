// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BuildControl /> should render 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <BuildControl />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="Label"
        />,
        <button
          className="Less"
        >
          Less
        </button>,
        <button
          className="More"
        >
          More
        </button>,
      ],
      "className": "BuildControl",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": undefined,
          "className": "Label",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Less",
          "className": "Less",
          "disabled": undefined,
          "onClick": undefined,
        },
        "ref": null,
        "rendered": "Less",
        "type": "button",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "More",
          "className": "More",
          "onClick": undefined,
        },
        "ref": null,
        "rendered": "More",
        "type": "button",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="Label"
          />,
          <button
            className="Less"
          >
            Less
          </button>,
          <button
            className="More"
          >
            More
          </button>,
        ],
        "className": "BuildControl",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": undefined,
            "className": "Label",
          },
          "ref": null,
          "rendered": null,
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Less",
            "className": "Less",
            "disabled": undefined,
            "onClick": undefined,
          },
          "ref": null,
          "rendered": "Less",
          "type": "button",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "More",
            "className": "More",
            "onClick": undefined,
          },
          "ref": null,
          "rendered": "More",
          "type": "button",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
